Automattic/jetpack
# AMP: do not add sharing CSS when Sharing module is not active

Fixes #22366 again

#### Changes proposed in this Pull Request:

* See https://github.com/Automattic/jetpack/pull/22370#issuecomment-1058320382

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Start a new site with both the AMP plugin and this branch.
* Go to Jetpack and connect the plugin to WordPress.com
* Once connected, go to `wp-admin/admin.php?page=jetpack_modules` and disable all modules, including sharing.
* Go to AMP > Settings.
* Switch between different template modes, and for each mode:
    * On the frontend, load a single post's view.
    * Click on the "AMP" button in the admin bar to load the AMP view.
    * You should never see the `social-logos.eot` file loaded on the page.
* Go back to `wp-admin/admin.php?page=jetpack_modules` and enable the Sharing feature.
* Re-do the above:
    * The sharing buttons should look good in every template mode.
